exercises/custom-set/example.py

```diff
@@ -0,0 +1,93 @@
+class CustomSet(object):
+    def __init__(self, *numbers):
+        self.size = 0
+        self.capacity = 100
+        num_buckets = int(self.capacity / 10)
+        self.data = [[] for x in range(num_buckets)]
+        for number in numbers:
+            self.add(number)                                                                                                                                                                                                                                                                                                                                                                              
+
+    def empty(self):
+        return self.size == 0
+    
+    def __len__(self):
+        return self.size
+
+    def expand(self):
```

Comment:
I think the hole `capacity` makes this implementation too complicated and is not needed in the set context of this exercise.